src/core/customRenderer/descriptors/CustomDescriptor.ts

```diff
@@ -95,11 +96,13 @@ export abstract class CustomDescriptor< //
   }
 
   public checkPropTypes(props: TProps, type: string): any {
-    checkPropTypes(this.propTypes,
-      props,
-      "prop",
-      type,
-      ReactDebugCurrentFiber.getCurrentFiberStackAddendum);
+    if (isNonProduction) {
+      checkPropTypes(this.propTypes,
+        props,
+        "prop",
+        type,
+        React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactDebugCurrentFrame!.getStackAddendum);
```

Comment:
`!`is a non-null assertion operator (https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator). `ReactDebugCurrentFrame` is only set when not in production and so it's an optional parameter on the `ReactSharedInternals` interface. However, I check to make it's nonProduction further up, so I can just assert that it's not undefined because I know it will be set if it's not in production.

Reply:
Nice :)